stem/boundary.py

```diff
@@ -0,0 +1,134 @@
+from typing import List, Dict, Any, Union, Optional
+from dataclasses import dataclass, field
+from abc import ABC, abstractmethod
+
+
+@dataclass
+class BoundaryParametersABC(ABC):
+    """
+    Abstract base class for boundary parameters
+    """
+
+    pass
+
+    @property
+    @abstractmethod
+    def is_constraint(self) -> bool:
+        """
```

Comment:
Property which indicates if boundary condition is a constraint. 
 
 The fact that we later use it to check if it should be in a list of loads or constraints, is irrelevant for the user. Thats a pure Kratos IO thing